apps/pomodo/pomodoro.js

```diff
@@ -25,12 +25,12 @@ class State {
 
   clear () {
     clearWatch();
-    g.clear();
```

Comment:
Just for the future, I believe just `g.clearRect(Bangle.appRect)` will work here too

Reply:
Changed to that!